test/e2e/cluster_samples_operator_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	samplesapi "github.com/openshift/cluster-samples-operator/pkg/apis/samples/v1"
 	sampopclient "github.com/openshift/cluster-samples-operator/pkg/client"
 	sampleclientv1 "github.com/openshift/cluster-samples-operator/pkg/generated/clientset/versioned"
-	"github.com/openshift/cluster-samples-operator/pkg/operatorstatus"
+	operator "github.com/openshift/cluster-samples-operator/pkg/operatorstatus"
```

Comment:
This was changed from operatorstatus to operator, but there were no other changes? seems suspicious ...

Reply:
s/samplesapi/samplesapiv1/
s/sampopclient/samplesopclient/

Reply:
that also was a golang 1.11 / vscode thing

Reply:
there might be a setting that you can change to not do that.

Reply:
good point @coreydaley ... I'll hunt for that next time

given the current set of green tests I'm thinking of eschewing generating a new commit and risking flakes :-)